src/plays/expanding-cards/Readme.md

```diff
@@ -0,0 +1 @@
+# expanding-cards
```

Comment:
Post the explanation of the play and what technical things it covers. See other play read me files.